test/browser/index.js

```diff
@@ -3,12 +3,14 @@ import sinonChai from 'sinon-chai';
 
 chai.use(sinonChai);
 
-import 'babel-polyfill';
+import 'core-js';
+import 'regenerator-runtime';
 
 import '../fixtures'; // import fixtures
 
 // import tests
 import './component';
 import './router';
+import './component-param';
```

Comment:
```suggestion
import './param-component';
```
(also fits with my naming suggestion above)